src/js/controllers/chatController.js

```diff
@@ -62,51 +62,46 @@ module.exports = ViewController.extend({
     },
 
     sendMessage: function(text) {
-        return $.Deferred().resolve().then(function() {
-            var promise = $.Deferred();
-
-            if (this.conversation.isNew()) {
-                this.conversation = this.collection.create(this.conversation, {
-                    success: promise.resolve,
-                    error: promise.reject
-                });
-            } else {
-                promise.resolve(this.conversation);
-            }
-
-            return promise;
-        }.bind(this))
```

Comment:
Babel polyfills *everything* to make it on par with a fully ES6 compliant environment. It's a bit overkill if you don't need everything.